README.id.md

```diff
@@ -0,0 +1,71 @@
+# sway
+
+[**English**](https://github.com/swaywm/sway/blob/master/README.md#sway--) - [日本語](https://github.com/swaywm/sway/blob/master/README.ja.md#sway--) - [Deutsch](https://github.com/swaywm/sway/blob/master/README.de.md#sway--) - [Ελληνικά](https://github.com/swaywm/sway/blob/master/README.el.md#sway--) - [Français](https://github.com/swaywm/sway/blob/master/README.fr.md#sway--) - [Українська](https://github.com/swaywm/sway/blob/master/README.uk.md#sway--) - [Indonesian](https://github.com/swaywm/sway/blob/master/README.id.md#sway--) - [Italiano](https://github.com/swaywm/sway/blob/master/README.it.md#sway--) - [Português](https://github.com/swaywm/sway/blob/master/README.pt.md#sway--) -
+[Русский](https://github.com/swaywm/sway/blob/master/README.ru.md#sway--) - [Български](https://github.com/swaywm/sway/blob/master/README.bg.md#sway--)
+
+sway adalah komposer untuk i3 [Wayland](http://wayland.freedesktop.org/).
+Pertanyaaan dan jawaban umum [FAQ](https://github.com/swaywm/sway/wiki). Ikut [channel
+ IRC](http://webchat.freenode.net/?channels=sway&uio=d4) (#sway on
+irc.freenode.net).
+
+Jika ingin mendukung perkembangan sway, kunjungi [Halaman Pateron SirCmpwn](https://patreon.com/sircmpwn).
+
+## Tanda Tangan Release
+
+Tiap Release ditandatangani dengan [B22DA89A](http://pgp.mit.edu/pks/lookup?op=vindex&search=0x52CB6609B22DA89A)
+dan diterbitkan [di GitHub](https://github.com/swaywm/sway/releases).
+
+## Instalasi
+
+### Dari Packages
+
+Sway tersedia dalam beberapa distribusi. Lakukan instalasi dengan package "sway" terlebih dahulu. 
+Apabila gagal, lihatlah [halaman wiki](https://github.com/swaywm/sway/wiki/Unsupported-packages)
+untuk informasi instalasi untuk tiap distribusi.
+
```

Comment:
Sway tersedia untuk banyak distibusi Linux, coba di instal package "sway" untuk distibusi anda, kalau tidak tersedia untuk distibusi anda, coba baca [Wiki] untuk informasi menginstal di distibusi anda.

Reply:
Agree with @x8BitRain version, but there are grammatical errors. I think it should be:

Sway tersedia untuk banyak distribusi Linux, coba instal package "sway" untuk distribusi Anda, kalau tidak tersedia, coba baca [Wiki] untuk informasi instalasi di distribusi Anda.

Alternatively, if "Anda" is too formal, just use "kamu". If you want to use "Anda", that word must be capitalized.

Reply:
Went with, ha1zum's suggestion. Also changed all "anda" to"Anda". Thanks!